changelog.d/20241203_224257_pjhinton_sc_29242_endpoint_search_filter_entity_type.md

```diff
@@ -0,0 +1,3 @@
+### Enhancements
+
+* Added a --filter-entity-type option on endpoint search.
```

Comment:
```suggestion
* Added a `--filter-entity-type` option on endpoint search.
```

Small tweak. Since this renders on the doc site it's nice to give code-style.